model/src/main/proto/spine_examples/shareaware/wallet/wallet.proto

```diff
@@ -54,13 +55,16 @@ message Wallet {
     //
     spine.money.Money balance = 2;
 
-    // An amount of reserved money.
+    // The binding of `WithdrawalId` to the amount of money reserved by it.
     //
-    // This amount is considered reserved, because there may be pending operations against this wallet,
-    // that may imply spendings. It is not possible to use the reserved amount directly,
-    // other than in scope of these pending operations.
+    // Many withdrawal operations may interact with this wallet
+    // and make money reservations for their needs.
+    // The amount of money is considered reserved because no one can use this amount
+    // except the operation that reserved it.
     //
-    spine.money.Money reserved_money = 3;
+    // The first generic in the `reserved_money` map is meant to be the `WithdrawalId` value.
```

Comment:
I think we need to be more precise here; in fact, its `WithdrawalId.uuid` value (if I am not mistaken).